web/service/src/main/java/org/mobilitydata/gtfsvalidator/web/service/Main.java

```diff
@@ -0,0 +1,233 @@
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import org.mobilitydata.gtfsvalidator.input.CountryCode;
+import org.mobilitydata.gtfsvalidator.runner.ValidationRunner;
+import org.mobilitydata.gtfsvalidator.runner.ValidationRunnerConfig;
+import org.mobilitydata.gtfsvalidator.util.VersionResolver;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+@SpringBootApplication
+@RestController
+public class Main {
+
+  static final String userUploadBucketName = "gtfs-validator-user-uploads";
+  static final String resultsBucketName = "gtfs-validator-results";
+  static final String jobInfoBucketName = "gtfs-validator-results";
+  final String fileName = "gtfs-job.zip";
+  static final String jobFilenamePrefix = "job";
+  static final String jobFilenameSuffix = ".json";
+  static final String jobFilename = jobFilenamePrefix + jobFilenameSuffix;
+  static final String countryCodeKey = "country-code";
+  static final String defaultCountryCode = "US";
+  static final String tempFolderName = "gtfs-validator-temp";
+
+  @Getter(AccessLevel.PROTECTED)
+  @Setter(AccessLevel.PROTECTED)
+  @Autowired
+  private Storage storage;
+
+  private static ConfigurableApplicationContext applicationContext;
+
+  public static void main(String[] args) {
+    applicationContext = SpringApplication.run(Main.class, args);
+  }
+
+  public static String getJobInfoPath(String jobId) {
+    return jobId + "/" + jobFilename;
+  }
+
+  public static File getRemoteFile(String remoteLocation, String localPrefix, String localSuffix)
+      throws Exception {
+    try {
+      var tempDir = Files.createTempDirectory(tempFolderName).toFile();
+
+      var inputResource = applicationContext.getResource(remoteLocation);
+      var inputFile = inputResource.getInputStream();
+
+      var tempFile = File.createTempFile(localPrefix, localSuffix, tempDir);
+      var output = new FileOutputStream(tempFile);
+      inputFile.transferTo(output);
+
+      return tempFile;
+    } catch (Exception exc) {
+      System.out.println("Error could not load remote file:" + exc.toString());
+      throw exc;
+    }
+  }
+
+  public void setJobCountryCode(String jobId, String countryCode) throws Exception {
+    try {
+      var jobInfoPath = getJobInfoPath(jobId);
+      var jobBlobId = BlobId.of(jobInfoBucketName, jobInfoPath);
+      var jobBlobInfo = BlobInfo.newBuilder(jobBlobId).setContentType("application/json").build();
+      var om = new ObjectMapper();
```

Comment:
Instead of serializing key-value pairs with ObjectMapper, could we introduce a simple object here with the appropriate fields and serialize that with Jackson/GSON/whatever instead?  I think it would make the code clearer about what's going on.  Something like `JobMetadata`?  And I'd also rename these methods as setJobMetadata and getJobMetadata because I think there will likely be more things we'll want to store here long term.